jut-server.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
-DIR="$( cd "$( dirname "$0" )" && pwd )"
-CMD="$DIR/target/appassembler/bin/jut-server"
+CMD="target/appassembler/bin/jut-server"
```

Comment:
This does not seem to be related to the PR and reverts the original improvement of being able to run it from different directory. Why?